### DISCUSSION TITLE
genomic data integration question

### DISCUSSION THREAD

**Discussion by newbiologist** [opensafely/documentation#1377] [Q&A] [answered]
Hello folks,

Is it possible to analyze genomic data using OpenSAFELY? Or does it only work with Electronic Health Record (EHR) data and other data mentioned on this page?
Data Sources - OpenSAFELY documentation
https://docs.opensafely.org/data-sources/

**Comment by evansd** [comment, marked as answer]
It would certainly be possible _in principle_ to do so: nothing in the fundamental design of the platform restricts it to working with just EHR data. But we've not worked with genomic data so far, and I don't know of any plans to do so in the immediate future.
